tdrs-backend/tdpservice/search_indexes/models.py

```diff
@@ -12,12 +15,12 @@ class T1(models.Model):
     Mapped to an elastic search index.
     """
 
-    record = models.CharField(max_length=156, null=False, blank=False)
-    rpt_month_year = models.IntegerField(null=False, blank=False)
-    case_number = models.CharField(max_length=11, null=False, blank=False)
-    disposition = models.IntegerField(null=False, blank=False)
-    fips_code = models.CharField(max_length=100, null=False, blank=False)
+    # def __is_valid__():
```

Comment:
`is_valid` can look across multiple fields and we can implement custom logic whereas `full_clean` is per-field validation via model/db validators at the end of declaration.